python/spacewalk/satellite_tools/repo_plugins/deb_src.py

```diff
@@ -409,6 +412,11 @@ def list_packages(self, filters, latest):
             to_return.append(new_pack)
         return to_return
 
+    def _load_urlgrabber_logspec(self):
+        with cfg_component('server.satellite') as CFG:
+            if CFG.urlgrabber_logspec:
+                self.urlgrabber_logspec = CFG.urlgrabber_logspec
```

Comment:
I would suggest to move these logic directly to `set_download_parameters`. I don't see any reason to keep it in the distinct method. And the same for `yum_src` plugin.